app/assets/stylesheets/ama_layout/layout_components/sidebar.scss

```diff
@@ -117,7 +111,7 @@ span.nav-icon{
   margin: $base-margin*2;
   margin-top: $base-margin*4;
   font-size: $base-font-size*1.5;
-  color: $smoke;
+  color: rgb(179, 179, 179);
```

Comment:
I think smoke looks fine, its not that much lighter than the rgb value 
![image](https://cloud.githubusercontent.com/assets/1906696/10250418/ecb138ec-68e7-11e5-84b1-27b383c7de30.png)